eng/pipelines/templates/variables/globals.yml

```diff
@@ -1,6 +1,6 @@
 variables:
   DefaultOptions: '--batch-mode --fail-at-end -Dmaven.wagon.http.pool=false --settings eng/settings.xml'
   LoggingOptions: '-Dorg.slf4j.simpleLogger.defaultLogLevel=error -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
-  Agent.Source.Git.ShallowFetchDepth: 1
+  #Agent.Source.Git.ShallowFetchDepth: 1
```

Comment:
Did you want to try and up the depth as a first step to dealing with the intermittent failures?

Reply:
I think for now I just want to stop the bleeding. Since we made the gh-pages change clone times should be way better anyway.